javascript/builtins/JSON.json

```diff
@@ -161,6 +161,60 @@
               "deprecated": false
             }
           }
+        },
+        "json_superset": {
+          "__compat": {
+            "description": "JavaScript is a superset of JSON",
```

Comment:
I think the meaning of this is sufficiently obscure that it would be nice to have an `"mdn_url"` (when I first saw this, I thought, "wait, wasn't JavaScript _always_ a superset of JSON?" and then I learned something new). Maybe pointing to [that JavaScript and JSON differences page you linked to](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON#JavaScript_and_JSON_differences) would work?

Reply:
That’s the page on which this would go to, and the Compat Table generator doesn’t yet cope well with links that contain anchors (at least until https://github.com/mdn/browser-compat-toolkit/pull/5 gets merged).

That said, I agree that adding that link is a good idea.